src/ocamlorg_frontend/pages/home.eml

```diff
@@ -353,7 +353,7 @@ Layout.render
     <div class="flex flex-col lg:flex-row justify-between lg:space-x-20 space-y-10 lg:space-y-0">
       <div class="shadow-lg p-10 bg-legacy-default dark:bg-legacy-dark-default rounded-xl lg:mb-0">
```

Comment:
```suggestion
      <div class="shadow-lg p-10 bg-white dark:bg-dark-background rounded-xl lg:mb-0">
```

Reply:
@SaySayo I think I added this changes by mistake. I'll discard the changes I made to this file for now and create a separate pr for this page when there's a design for it.